src/components/SelectedFilterOptionList/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./index.css";
+
+// It renders the selected filter options of a given state which tracks the selected filter options.
+// It is called in AdvancedFilters component
+const FilterOptionsList = ({ selectedOptions }) => {
```

Comment:
new line here

Reply:
in line 7?

Reply:
if I add space in line 7, the tests don't pass

Reply:
I added space between line 5 and 6, in case you meant a new line there

Reply:
@fadime-ozdemir why tests fail when u add newline between 6 and 7 ?

Reply:
The tests are failing because the yarn run fix I think. I pushed with the extra space but when I saw the error it said I needed to run yarn fix, so I did, but that command deletes the extra space